lncfg/db.go

```diff
@@ -23,6 +23,8 @@ const (
 	PostgresBackend            = "postgres"
 	DefaultBatchCommitInterval = 500 * time.Millisecond
 
+	defaultPostgresMaxConnections = 50
```

Comment:
Should this be zero instead? (unlimited)

Reply:
I think that unlimited is a dangerous setting. It has the potential to cause errors under high load conditions and I am slightly concerned that there may exist error paths that leave lnd a sub-perfect state. With bbolt, the chances of running into that are very low, so I consider it mostly untested. How do you feel about it?

Reply:
Btw, 100 is the default postgres limit. Therefore the default of 50.

Reply:
> I think that unlimited is a dangerous setting. It has the potential to cause errors under high load conditions and I am slightly concerned that there may exist error paths that leave lnd a sub-perfect state. With bbolt, the chances of running into that are very low, so I consider it mostly untested. How do you feel about it?

SGTM